packages/devextreme-schematics/src/add-layout/files/src/dx-styles.scss

```diff
@@ -61,15 +58,14 @@ html, body {
   border-left: 0;
 }
 
-.with-footer > .dx-scrollable-wrapper >
-.dx-scrollable-container > .dx-scrollable-content {
+.with-footer>.dx-scrollable-wrapper>.dx-scrollable-container>.dx-scrollable-content {
   height: 100%;
 
-  & > .dx-scrollview-content {
+  &>.dx-scrollview-content {
```

Comment:
why these spaces changes happened? do you have other scss lint rules in your ide? i prefer old styles cause they are more readable